receiver/statsdreceiver/protocol/statsd_parser.go

```diff
@@ -181,6 +189,11 @@ func buildPoint(parsedMetric *statsDMetric) (*metricspb.Point, error) {
 			}
 			parsedMetric.metricType = metricspb.MetricDescriptor_GAUGE_INT64
 		})
+	case "ms":
+		return buildTimerPoint(parsedMetric, now, func(parsedMetric *statsDMetric) {
+			// we may need to change the transferred type in the future
```

Comment:
How about adding a TODO and filing an issue if there's something in-progress for this type?

Reply:
Sure, will add a TODO. Maybe no issue now since we plan to only use Double Gauge for timer. This line is for the future if we change our mind.